src/decode.rs

```diff
@@ -2,8 +2,8 @@ use crate::{CompoundTag, Tag};
 use byteorder::{BigEndian, ReadBytesExt};
 use flate2::read::{GzDecoder, ZlibDecoder};
 use linked_hash_map::LinkedHashMap;
-use std::io;
-use std::io::Read;
+use std::{error::Error, io::Read};
```

Comment:
Can you pls format with default formatting rules?

Reply:
Hm, I think I'm using the default settings. How does the result of rustfmt differ for you?